com.unity.netcode.gameobjects/Runtime/Messaging/Messages/NetworkTransformMessage.cs

```diff
@@ -132,7 +151,10 @@ public bool Deserialize(FastBufferReader reader, ref NetworkContext context, int
                         ownerClientId = context.SenderId;
                     }
 
-                    var networkDelivery = State.IsReliableStateUpdate() ? NetworkDelivery.ReliableSequenced : NetworkDelivery.UnreliableSequenced;
+                    // Depending upon whether it is spawned locally or not, get the deserialized state
```

Comment:
This is the 2nd part of the fix where State was never being set and it wasn't adjusting based on whether the host, server, or DAHost had a NetworkTransform instance.